.github/workflows/test.yml

```diff
@@ -21,14 +21,14 @@ jobs:
         fetch-depth: 0
 
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
 
     - name: Install dependencies
       run: |
-        sudo apt-get update
-        sudo apt-get install csh
+        sudo apt update
+        sudo apt install csh
```

Comment:
Nice, if I understand correctly apt and apt-get have the same behaviour, but apt is more user friendly.